features/steps/v6uatsteps.rb

```diff
@@ -0,0 +1,53 @@
+require 'date'
```

Comment:
I wouldn't name this file after the UAT.

Reply:
As discussed IRL to simplify the moving parts during testing these functions will be moved to maze-runner post UAT work.  They're collected here in a single very specifically named file to draw attention to this.

Reply:
Ok, fair enough, I guess it does increase the chances of us refactoring it sooner rather than later.